runtime/util/finish_rewriting.cpp

```diff
@@ -44,6 +45,10 @@ int32_t get_exit_code(block *);
     } else {
       print_configuration(output_file, subject);
     }
+  } else if (!error && !proof_hint_instrumentation_slow) {
+    write_uint64_to_file(output_file, 0xFFFFFFFFFFFFFFFF);
+    serialize_configuration_to_file(output_file, subject, true, false);
+    write_uint64_to_file(output_file, 0xCCCCCCCCCCCCCCCC);
   }
```

Comment:
I just didn't understand this part. Can you elaborate on this?

Reply:
I think what's happening here is that if `proof_hint_instrumentation_slow` is set, then the final configuration gets emitted by the final rewrite event. Without it, though, we still want to emit the final configuration, so we do that here. Is that correct, @theo25 ?

Reply:
yes that is correct. When the slow flag has been passed we don't need to emit the final config here because it would have been emitted the same way that the rest of the intermediate configs would have been emitted. But when the slow flag has not been passed, we need to emit it here.